cmd/pulumictl/dispatch/cli.go

```diff
@@ -53,7 +64,6 @@ func Command() *cobra.Command {
 			}
 
 			_, err := semver.Parse(gitversion.StripModuleTagPrefixes(ref))
```

Comment:
This semver check needs moving into the `payloadMap["ref"]` side of the if above.

Reply:
Linting didn't pick this up because there's global var defined at line 22. That can go but I'm pretty sure the others can probably go as well.

Reply:
Ah yes, good catch, thanks!

Yeah indeed, all the global variables can go.  In fact that made me realize that we never actually used the `org` variable, and that command line parameter doesn't do anything.  I removed it now.